### ISSUE TITLE
Create new release of silk

### ISSUE THREAD

**Issue by albertyw** [jazzband/django-silk#187]
I think once the jazzband migration is complete, it would be a good time to make a new release of silk, either manually or by setting up [automatic releases](https://jazzband.co/about/releases).

**Comment by avelis** [comment]
@albertyw Agreed on the new release once the migration is complete. Will look into possibly getting the Travis-CI config file to build on tagged releases.

**Comment by avelis** [comment]
@albertyw I have added the config to make releases on tags in the following commit: 53b5ea7e1b4940ada013985cbce009651f716b71

However, since this repo as moved to Jazzband there are some more administrative steps that need to occur.

**Comment by albertyw** [comment]
Is there an update on this?

**Comment by avelis** [comment]
@albertyw I [have commented on the need to add jazzband to access the silk pypi project in order to make releases through that account](https://github.com/jazzband/roadies/issues/67#issuecomment-320000561). I am not sure what else I can do but it's part of the requirements. Maybe we should make a release before the migration is completed?

**Comment by nias-r** [comment]
If there's some difficulty in getting everything in order with pypi and jazzband, my vote would be to make a final release under the existing setup.

**Comment by albertyw** [comment]
I'm going to try making a new release now that the pypi permissions issues should be fixed.

**Comment by avelis** [comment]
@albertyw The version value is in setup.py. If you want to break that out into a different mechanic that's fine. FYI It is still currently at `1.0.0`. If you decide to make a release make sure it gets updated. That has burned me a couple times.

**Comment by albertyw** [comment]
There's a bug in the travis config.  This was in the build output:
```
Fetching: dpl-1.8.43.gem (100%)
Successfully installed dpl-1.8.43
1 gem installed
invalid option "--password="
failed to deploy
```

Based on https://github.com/travis-ci/dpl/issues/327, it seems there's a problem with the encrypted password that was added in 53b5ea7e1b4940ada013985cbce009651f716b71.  I'm also not sure if we're under the "lead project mamber" system mentioned in https://github.com/jazzband/roadies/issues/69.  I created https://github.com/jazzband/roadies/issues/82 to have someone follow up.

**Comment by avelis** [comment]
@albertyw I added the encrypted password but it was done using deprecated documentation for TravisCI config setup. Looking at the info in jazzband/roadies#69 it appears a structural change is now in place for reviewing. I guess I missed all this.

Thanks for making jazzband/roadies#82

**Comment by yousifa** [comment]
Any update on release on pypi?

**Comment by avelis** [comment]
@yousifa Not that I know of.

**Comment by albertyw** [comment]
As an update to everyone here, https://github.com/jazzband/roadies/issues/82 has been resolved and I've created a temporary branch release-1.1.0 (do not merge into master) to get 1.1.0 on pypi (relevant discussion on https://github.com/jazzband/silk/commit/78ad261969418501164db11d49191c0b030103e8#commitcomment-26718482).  It's currently blocked by https://github.com/jazzband/website/issues/41 but can be retried by rerunning https://travis-ci.org/jazzband/silk/jobs/326440835.

**Comment by jezdez** [comment]
Okay, I've fixed https://github.com/jazzband/website/issues/41 but now the upload fails (correctly) with:

```
Uploading distributions to https://jazzband.co/projects/silk/upload
Uploading django_silk-1.1.0-py2.py3-none-any.whl
HTTPError: 400 Client Error: The filename for 'silk' must start with 'silk'. for url: https://jazzband.co/projects/silk/upload
```

The problem is basically that the repo and project are called "[silk"](https://pypi.python.org/pypi/silk) while I think you expect things to released to PyPI as "[django-silk](https://pypi.python.org/pypi/django-silk)". I'll go ahead and fix the name of the repo and project on jazzband.co so this won't happen again.

**Comment by jezdez** [comment]
Okay done. Any link to the GitHub project will automatically forward to the new name btw. I've restarted the [Travis build](https://travis-ci.org/jazzband/silk/jobs/326440835) to do the release.

**Comment by xrmx** [comment]
Build passed but there's HTTPError: 404 Client Error: NOT FOUND for url: https://jazzband.co/projects/silk/upload in the travis logs

**Comment by jezdez** [comment]
Yep, just fixed in https://github.com/jazzband/django-silk/commit/de08d5b538144d540306bc029d65c45ceed6f580. Build runs in https://travis-ci.org/jazzband/silk/builds/327171966.

**Comment by jezdez** [comment]
BTW, the gitter.im links didn't seem to work, what's up with that?

**Comment by jezdez** [comment]
Gnaah, apparently Travis-CI didn't handle the repo rename nicely (maybe because GitHub's API is acting up today and the webhook didn't work?), even though I've seen it handle repo renames nicely in the past for other projects. I've just re-enabled travis for the newly named django-silk repo and will restart the build again.

**Comment by jezdez** [comment]
Aha, one of the builds for https://travis-ci.org/jazzband/django-silk/builds/327171966 was still running and apparently released correctly to Jazzband. @avelis you should have gotten an email to confirm the release.

**Comment by jezdez** [comment]
Oh, @albertyw did you want to be able to be lead project member as well as @avelis?

**Comment by avelis** [comment]
@jezdez I appreciate all your diligence in getting all the tracks setup for releasing through Jazzband. I did get the confirmation email. I will confirm it later in the morning (currently 5:07am).

Gitter.im was functioning when I took over the repo a year ago but I haven’t been really active on it.

**Comment by avelis** [comment]
@jezdez I don't mean to delay this any further but I am getting a 2FA must be setup error.
> Sorry, but to release the upload you need to have Two Factor Auth (2FA) enabled on GitHub.
I just set it up again but it's still giving me the error. Am I missing this security setting somewhere else?

**Comment by albertyw** [comment]
@jezdez Yes, I'd like to be a project lead too.  I have 2FA auth set up for my account already so maybe I can get the release to work.

**Comment by avelis** [comment]
@jezdez @albertyw  Ok, it looks like worked now. Might have been a caching issue on Github's side.
Release 1.1.0 is out.
https://pypi.python.org/pypi/django-silk/1.1.0
